spec/tests_base_decorator_spec.rb

```diff
@@ -0,0 +1,19 @@
+require '.\base_decorator'
```

Comment:
- [ ] Kindly use correct path in your import.This should be `./base_decorator` NOT `.\base_decorator`. When I test your app with this wrong path, almost your files failed. Kindly check other files and correct the path when needed.

### error
![unit-erro](https://github.com/ibugithub/School-Library/assets/26597432/df9a53d0-73ce-4052-9065-574be66df98a)

Reply:
✔️ Changes done. Thank you. 👍🏽